packages/unchained-client/src/evm/polygon/parser/__tests__/mockData/tokens.ts

```diff
@@ -0,0 +1,22 @@
+import type { Token } from '../../../../../types'
+
+export const maticToken: Token = {
+  contract: '0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0',
+  decimals: 18,
+  name: 'Matic',
+  symbol: 'MATIC',
```

Comment:
The parser is a Polygon Tx parser, and this is the address of MATIC on Ethereum, is this intended?

Reply:
fixed, there was some confusion with native assets and tokens